### DISCUSSION TITLE
Why there is no Mac version?

### DISCUSSION THREAD

**Discussion by thisisjamaldin** [TokisanGames/Terrain3D#179] [Questions] [answered]
Wanna try last versions (with a bunch of features) but can't find mac version, while trying to download from "Assets"

**Comment by TokisanGames** [comment, marked as answer]
It works on mac if you build it yourself. I don't have a mac on which to build. It is possible to set up a cross compiling toolchain, but it's extensive and complicated. It's much simpler if you just build it on your own system. The process is well documented and easy to do. See the wiki. 

I do post any community provided builds if we receive any.

**Reply by thisisjamaldin** [reply]
https://github.com/thisisjamaldin/terrain3d_0.8.2-alpha_gd4.1.1_osx

Did it, Thanks)
I compressed the "project" folder as "terrain3d_0.8.2-alpha_gd4.1.1_osx.zip"

**Reply by TokisanGames** [reply]
Thank you. Is this for x86 or arm processors? I've added it to the release.

**Reply by thisisjamaldin** [reply]
intel i5

**Reply by TokisanGames** [reply]
I'm experimenting with using Github's CI to build macos. The Github docker is an x64 system however observing the build command line I noticed it has keywords like `..._debug.universal.o ... -arch x86_64 -arch arm64`, which makes me think these binaries might work on both M1 and x64 architectures. Can I get some testers? Please download this and see if it works and let me know which architecture you have.

Download the artifact here:
https://github.com/TokisanGames/Terrain3D/actions/runs/6030109506
